src/app/atoms/SeedPhraseInput.tsx

```diff
@@ -153,18 +151,19 @@ export const SeedPhraseInput: FC<SeedPhraseInputProps> = ({
           />
         </div>
       </div>
+
       {!isFirstAccount && <div className="text-xs font-medium text-red-600 text-center">{labelWarning}</div>}
+
       {!isFirstAccount && (
         <div className="text-xs font-medium text-red-600 text-center mb-6">
           <T id="seedPhraseAttention" />
         </div>
       )}
-      <div
-        className={classNames('w-full text-center', 'pb-2 mb-6', 'text-gray-700', 'border-b-2')}
-        style={{ borderBottomWidth: 1 }}
-      >
+
+      <div className="w-full text-center pb-2 mb-6 text-gray-700 border-b-2" style={{ borderBottomWidth: 1 }}>
```

Comment:
instead of `style={{ borderBottomWidth: 1 }} `just add 'border-b' to classNames 
and also you have `border-b-2` className, that means borderBottomwidth - 2, you should choose one what you need